src/app/entity-groups/journal-entities/item-pages/journal-issue/journal-issue.component.html

```diff
@@ -1,3 +1,9 @@
+<div class="row" *ngIf="showBackButton | async">
+  <div class="col-12">
+    <ds-themed-results-back-button></ds-themed-results-back-button>
+  </div>
+</div>
+
```

Comment:
Perhaps you can move this `showBackButton`  check to `ds-results-back-button` itself? that way there'd be less code duplication. Since you're only using a `col-12` (i.e. a full width `col`) you should be able to just leave out the `row` and `col` divs without seeing a difference